### ISSUE TITLE
Feature request: swipe to remove bookmarks

### ISSUE THREAD

**Issue by FolkyHobbit** [PaulWoitaschek/MaterialAudiobookPlayer#612]
I think it would be nice to be able to swipe to remove a bookmark. I save a bookmark with the sleep timer, but over time this means I collect tens of them. So sometimes I want to do a cleaning but it takes 3 clicks per bookmark now.

**Comment by illarionds** [comment]
+1 - it becomes a bit of a chore clearing them all.

A "clear all except latest" would also solve the problem perfectly for me (but perhaps not for others, if they actually set bookmarks they want to keep).

"Clear all bookmarks set by setting sleep timer" would also do it :)

**Comment by FolkyHobbit** [comment]
Yes all of these solutions would also solve my problems. I suggested the swipe solution because I felt it fits best with the philosophy of the app of keeping things simple and avoiding unnecessary menu items. Clearing them all would be nice though but maybe we need different categories of bookmarks so you can only clean out the sleep timer ones. (but now things are getting more complicated...)

**Comment by FolkyHobbit** [comment]
I forgot about this suggestions I made a while ago, but now I tested it and it was implemented! Thanks for that! Shall I close the issue or was it left open for some reason?

**Comment by nioncode** [comment]
Should be closed as it is already implemented.
